src/app/client/src/app/modules/workspace/components/data-driven/data-driven.component.html

```diff
@@ -1,43 +1,64 @@
-<sui-modal
-    [mustScroll]="true"
-    [isClosable]="false"
-    [transitionDuration]="0"
-    [size]="'normal'"
-    class="sb-modal"
-    appBodyScroll
-    (dismissed)="goToCreate();"
-    #modal
-    [appTelemetryImpression]="telemetryImpression">
+<sui-modal [mustScroll]="true" [isClosable]="false" [transitionDuration]="0" [size]="'normal'"
+  class="sb-modal sb-course-modal" appBodyScroll (dismissed)="goToCreate();" #modal
+  [appTelemetryImpression]="telemetryImpression">
 
-    <i class="icon close" (click)="modal.deny('denied')" tabindex="0"></i>
+  <i class="icon close" (click)="modal.deny('denied')" tabindex="0"></i>
 
-    <!--Header-->
-    <div class="sb-modal-header">
-        {{resourceService?.frmelmnts?.lbl[creationFormLable]}}
-    </div>
-    <!--/Header-->
+  <!--Header-->
+  <div class="sb-modal-header">
+    What do you want create?
```

Comment:
do not remove the label here {{resourceService?.frmelmnts?.lbl[creationFormLable]}}, the new label should come in a certain condition, better add one separate header section.